app/views/tasks/index.html.erb

```diff
@@ -0,0 +1,14 @@
+<%= render partial: "tasks_all_add" %>
+
+<div id="tasks">
+  <h2> Your Tasks</h2>
+    <ol>
+      <% @tasks.each do |task| %>
+        <li id="task-name">
+          <%= render partial: "task_summary", locals: {task: task} %>
+        </li>
+          <%= render partial: "task_complete_edit_delete", locals: {task: task} %>
```

Comment:
Because both the `task_complete_edit_delete` and `task_summary` partials are only used in this location we could simplify things by putting the code for them directly inside of this view file.